spec/models/manageiq/providers/embedded_terraform/automation_manager/configuration_script_source_spec.rb

```diff
@@ -85,10 +99,11 @@ def files_in_repository(git_repo_dir)
         expect(names.first).to(eq("/hello_world_local"))
 
         expected_hash = {
-          "relative_path" => File.dirname(*repo_dir_structure),
-          "files"         => [File.basename(*repo_dir_structure)],
-          "input_vars"    => nil,
-          "output_vars"   => nil
+          "relative_path"     => File.dirname(*repo_dir_structure),
+          "files"             => [File.basename(*repo_dir_structure)],
+          "input_vars"        => @hello_world_variables_response['template_input_params'],
```

Comment:
If you need to use `@hello_world_variables_response` recommend using `let()` instead of an instance-variable:
```
let(:hello_world_response) { JSON.parse(File.read(File.join(__dir__, "runner/data/responses/hello-world-variables-success.json"))) }
```

Then you can just do this here:
```suggestion
          "input_vars"        => hello_world_response['template_input_params'],
```